src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/.editorconfig

```diff
@@ -0,0 +1,148 @@
+[*.cs]
```

Comment:
We should just have a single .editorconfig for the whole sol'n. I think we should make a separate PR into main that adds one